format/abc/Data.hx

```diff
@@ -158,8 +159,8 @@ class ABCData {
 	public var inits : Array<Init>;
 	public var functions : Array<Function>;
 
-	public function get<T>( t : Array<T>, i : Index<T> ) : T {
-		return switch( i ) { case Idx(n): t[n-1]; };
+	public inline function get<T>( t : Array<T>, i : Index<T> ) : T {
+		return t[i.asInt() - 1];

```

Comment:
Not related to this PR, but in my code, to get a correct value from `methodTypes` I had to write another `get` function that doesnt do `- 1`. 

Also, Ideally we should also wrap ABCData's arrays into abstracts with nice `get` methods that handle this, so one can do `abc.namespaces.get(nsIndex)` instead of `abc.get(abc.namespaces, nsIndex)`. I can do that if you agree.